README-LICENSE.md

```diff
@@ -0,0 +1,63 @@
+
+# Licensing
+
+Unless noted below, NeoForge and its all parts here in this repository are
+licensed under the **GNU Lesser General Public License v2.1** (LGPL v2.1), as 
+seen  at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt and reproduced
```

Comment:
```suggestion
seen at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt and reproduced
```

Reply:
It used to say there "seen in its canonical location at", but I removed that phrase since it pushed the URL to the next line and left a large gap. 😆